Maintenance for the Serbo-Croatian Filters section

DandelionSprout opened the pull request:
- Added 2 CDNs, one of which lead to GitLab as is the case for Nordic Filters (Last night's temporary takedown for a few hours of the AdGuardFilters GitHub repo, made me extra aware of the need for non-GitHub backup hostings).
- Firefox's download pages list Bosnian as a distinct option, so I added their download page's ISO code for Bosnian to the "lang" values.

While I remember it's been wished for before to make threads for it in uBlock-issues instead of PRs, to make more people able to comment on suggested changes of mine, this one should on paper be simple enough to not need such a thread… I think.

DandelionSprout committed:
Maintenance for the Serbo-Croatian Filters section

gorhill commented:
If you want fallback URLs, best to add them to `contentURL`, i.e. convert to an array and append the URLs to the existing `contentURL` one. Using `cdnURLs` in your case will cause the relatively stale URLs you specified to always be used in lieu of the more reliable `raw.githubusercontent.com` one. The fact that your list does not use a `! Last-Modified: ` field means uBO's auto-update code will be unable to tell whether the list downloaded from one of the new URLs is more recent than the one currently cached. For cases of fallback URLs, they need to be appended to `contentURL`.

DandelionSprout commented:
I guess I'll need to do some changes to Nordic Filters too in that case, which has many of the same shortcomings in both assets.json and plausibly within the list itself.

DandelionSprout committed:
Implemented the request to move to contentURLs

I sure I hope I understood the request correctly, or things would get slightly awkward for me.
